src/CMakeLists.txt

```diff
@@ -1,12 +1,26 @@
 include_directories(ssplit)
 add_library(ssplit STATIC ssplit/ssplit.cpp ssplit/regex.cpp)
-set_target_properties(ssplit PROPERTIES LIBRARY_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}")
-set_target_properties(ssplit PROPERTIES ARCHIVE_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}")
+target_link_libraries(ssplit ${EXT_LIBS})
 
-if (SSPLIT_STANDALONE_COMPILE)
+set_target_properties(ssplit PROPERTIES
+  LIBRARY_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}"
+  ARCHIVE_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}")
+
+if (NOT COMPILE_LIBRARY_ONLY)
   include_directories(3rd-party/CLI11)
-  add_executable(ssplit_executable command/ssplit_main.cpp)
-  target_link_libraries(ssplit_executable ssplit ${EXT_LIBS} ${ABSEIL_LIBS})
-  set_target_properties(ssplit_executable PROPERTIES OUTPUT_NAME
-    "${CMAKE_BINARY_DIR}/ssplit")
-endif(SSPLIT_STANDALONE_COMPILE)
+  if (NOT WIN32)
+    # ssplit-main has some mmap() code, and include "sys/mman.h"
```

Comment:
ssplit-main is what tests the source right? So we're not testing windows? What happened here?

Reply:
Yes, we're currently not testing on Windows. The Ubuntu runners run tests.